vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -149,8 +181,169 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
+
+   ```
+    Navigation(
+        selector=Accordion(pages={"Title": ["My first page", "My second page"]})
+    )
+   ```
+#### Accordion examples
+
+- Simple Accordion
+
+<table>
+<tr>
+<td><h3 style="width:350px" align="center"><img align="center" height="450" width="350" src="../../../assets/user_guides/navigation/simple_nav.png"></h3></td>
+<td style="width:360px">
+
+    ```py
+    vm.Navigation(
+        selector=vm.Accordion(
+            pages=[
+                "My first page",
+                "My second page"
+            ]
+        )
+    )
+    ```
+
+</td>
+</tr>
+</table>
+
+- Customized Accordion
+
+<table>
+<tr>
+<td><h3 style="width:350px" align="center"><img align="center" height="450" width="350" src="../../../assets/user_guides/navigation/simple_nav.png"></h3></td>
+<td style="width:360px">
+
+    ```py
+    vm.Navigation(
+        selector=vm.Accordion(
+            pages={"Accordion title": [
+                "My first page",
+                "My second page"
+            ]}
+        )
+    )
+    ```
+
+</td>
+</tr>
+</table>
+
+### NavBar
+To add navigation bar with icons to your dashboard provide [`NavBar`][vizro.models.NavBar] model to the `selector` argument of the `Navigation`. 
+
+NavBar model has two arguments `pages` and `items`. 
+
+- **`NavBar.pages`**
+
+Use `pages` for default icon image and configuration. 
+   ```
+    Navigation(
+        selector=NavBar(pages=["My first page", "My second page"])
+    )
+   ```
+
+- **`NavBar.items`**
+
+Use `items` to add customization to the icons. `NavBar.items` is provided as list of `NavItem` models. `NavItem` model is responsible for building individual icons within navigation bar. 
+
+For further customizations of icons, you can refer to the [`NavItem`][vizro.models.NavItem] reference. Some popular choices are:
+
+- customize icon image by providing name of the icon from the [Google Material Icon library](https://fonts.google.com/icons) to `icon` argument of `NavItem` model
+- provide text string to be displayed in the icon tooltip on hover by using `text` argument of `NavItem` model
+
+```
+Navigation(
+    selector=NavBar(
+        items=[
+            vm.NavItem(
+                pages=["My first page"], icon="dashboard", text="Pages"
+            ),
+            vm.NavItem(
+                pages=["My second page"], icon="summarize", text="Summary"
+            ),
+        ]
+    )
+)
+```
+
+!!! tip
+    If you would like to turn off icon images you can specify the empty string `""` in the `icon` argument.
+
+#### NavBar examples
+
+- Navigation icons
+<table>
+<tr>
+<td><h3 style="width:350px" align="center"><img align="center" height="450" width="350" src="../../../assets/user_guides/navigation/table_nav1.png"></h3></td>
+<td style="width:360px">
+
+    ```py
+    
+    vm.Navigation(
+        selector=vm.NavBar(
+            pages={
+                "First icon": [
+                    "My first page", 
+                    "My second page"
```

Comment:
The screenshot on the left shows "My third page" instead of "My second page"
![Screenshot 2023-11-10 at 1 29 17 PM](https://github.com/mckinsey/vizro/assets/31973683/31a166c1-bcae-4bea-9cb7-5c552ebf572e)